lib/Service/ModelService.php

```diff
@@ -79,9 +97,9 @@ public function modelFileExists(int $modelId, string $file): bool {
 	 */
 	public function prepareModelFolder(int $modelId) {
 		try {
-			$this->appData->getFolder('/models/' . $modelId);
```

Comment:
Don't use root folder.. It can be literally any directory that has the correct permissions.

So, revert and add an:
```
if (!is_null($modelPath)) {
    mkdir($modelPath)
    return;
}
```

Reply:
```
if (!is_null($modelPath)) {
    if (file_exists($modelPath)) {
        mkdir($modelPath)
    }
    return;
}
```

MMM.. 
On second thought, to avoid errors ..